packages/console/src/pages/ApiResourceDetails/components/ManagementApiNotice/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Trans, useTranslation } from 'react-i18next';
+
+import InlineNotification from '@/ds-components/InlineNotification';
+import TextLink from '@/ds-components/TextLink';
+import useConfigs from '@/hooks/use-configs';
+import useDocumentationUrl from '@/hooks/use-documentation-url';
+
+function ManagementApiNotice() {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { getDocumentationUrl } = useDocumentationUrl();
+  const { configs, isLoading, updateConfigs } = useConfigs();
+  const { managementApiAcknowledged } = configs ?? {};
```

Comment:
can use `useUserPreferences` hook to avoid fallbacks?

Reply:
yes, updated